asv_bench/benchmarks/io/csv.py

```diff
@@ -53,6 +53,14 @@ class TimeReadCsvSkiprows(BaseReadCsv):
         UNARY_OP_DATA_SIZE[ASV_DATASET_SIZE],
         [
             None,
+            # integer value
+            min(
+                [
+                    UNARY_OP_DATA_SIZE[ASV_DATASET_SIZE][shape_id][0]
+                    for shape_id in range(len(UNARY_OP_DATA_SIZE[ASV_DATASET_SIZE]))
+                ]
+            )
+            // 10,
```

Comment:
Please make changes in ASV always in separate PR.

Reply:
Ok, ASV changes were removed.